projects/com.oracle.truffle.llvm.runtime/src/com/oracle/truffle/llvm/runtime/floating/LLVM80BitFloat.java

```diff
@@ -626,4 +642,108 @@ public static LLVM80BitFloat fromString(String stringValue) {
         return fromBytesBigEndian(DatatypeConverter.parseHexBinary(stringValue));
     }
 
+    protected abstract static class LLVM80BitFloatNativeCallNode extends LLVMNode {
```

Comment:
These changes should allow deleting quite some code in this file (e.g. the `add` implementation, replace the `compare` code). Can you remove dead code?